app/views/school/chromebooks/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<%- title = t('page_titles.school_edit_chromebooks') %>
+<%- content_for :title, title %>
+<% content_for :before_content, govuk_link_to('Back', school_details_path, class: 'govuk-back-link') %>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <span class="govuk-caption-xl"><%= @school.name %></span>
+      <%= title %>
+    </h1>
+    <p class="govuk-body">
+      If you are going to order Google Chromebooks, we’ll need 2 pieces of information to configure them:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>the domain a school uses for its Google services</li>
+      <li>the school’s recovery email address</li>
+    </ul>
+    <p class="govuk-body">
+      We need this information so we can secure the devices.
+    </p>
+    <p class="govuk-body">
+      If you need help finding these details, email <%= govuk_link_to('COVID.TECHNOLOGY@education.gov.uk', 'mailto:COVID.TECHNOLOGY@education.gov.uk') %>
+    </p>
+
+    <%= form_for @chromebook_information_form, url: school_chromebooks_path, method: :patch do |f| %>
```

Comment:
How much of this is reusable/shared with the RB chromebook form?

Reply:
ooooh, as it turns out, all of it if I make one or two tweaks..... good call!